src/twinkie_console.c

```diff
@@ -74,31 +87,31 @@ void *control_d(void *p) {
   sleep(2);  // wait 2 second to allow the stream buffer to clear
   ret = write_twinkie_shell(t, start, 6);
   bool file_is_open = false;
-  do {
-    file_is_open = !file_is_open;
-    if (file_is_open) {
-      time(&timer);
-      tm = localtime(&timer);
-      /* alternatively read file name from something */
-      sprintf(file_name, "./%s/%d_%d_%d_%d_%d_%d.bin", dir_name,
-              tm->tm_year + 1900, tm->tm_mon, tm->tm_mday, tm->tm_hour,
-              tm->tm_min, tm->tm_sec);
-      file_open(file_name);
-    } else {
-      file_close();
```

Comment:
Oh I just noticed I don't send the shell commands correctly when opening and closing the files. I've pushed a patch to correctly turn the twinkie on and off. If you rebase off of my patch it should be more clear where to go from there.

Sorry about the confusion.

Reply:
NP